pandunia-loge.csv

```diff
@@ -552,7 +552,7 @@ chen fikse||||prefix||prefijo||||||||||||||etuliite (prefiksi)|przedrostek (pref
 chen ga||||assume (presuppose)||presuponer||||||||||||||edellyttää|zakładać (założyć)
 chen vide||||expect (anticipate, predict, forsee)||prever (esperar, pensar)|||||待ち受ける (見込む)|||||||||ennustaa (nähdä ennalta)|przewidzieć (spodziewać się)
 chen vide di||||expected||||||||||||||||ennustettu|spodziewany (przewidziany)
-chen yang di||||classic (classical)||clásico|||||古典的|||||||||klassinen|klasyczny
+chen yanga di||||classic (classical)||clásico|||||古典的|||||||||klassinen|klasyczny
```

Comment:
I guess it should be **yang** because it's a common word and I didn't notice that the main entry for **yang** was changed.

Reply:
yes, you're rite.